WireCalling/Sources/WireCallingUI/WireMeetings/MeetingsListView.swift

```diff
@@ -0,0 +1,181 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+package import SwiftUI
+import WireCallingDomain
+import WireDesign
+
+package struct MeetingsListView: View {
+
+    private typealias Strings = L10n.Localizable.WireMeetings.List
+
+    @ObservedObject private var viewModel: MeetingsListViewModel
+
+    package init(viewModel: MeetingsListViewModel) {
+        self.viewModel = viewModel
+    }
+
+    package var body: some View {
+        VStack {
+            Picker("", selection: $viewModel.selectedTab) {
+                ForEach(MeetingsListViewModel.Tab.allCases, id: \.self) { tab in
+                    Text(tab.title).tag(tab)
+                }
+            }
+            .pickerStyle(.segmented)
+            .padding(.horizontal, 16)
+            .padding(.top, 12)
+            .accessibilityIdentifier("meetingsListPicker")
+
+            content.frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .top)
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .top)
+        .background(ColorTheme.Backgrounds.surface.color)
+    }
+
+    @ViewBuilder private var content: some View {
+        List {
+            if viewModel.selectedTab == .next {
+                if !viewModel.ongoingMeetings.isEmpty {
+                    Section {
+                        ForEach(viewModel.groupedOngoing.first?.timeSlots.first?.meetings ?? [], id: \.id) { meeting in
```

Comment:
issue: I feel like the view model should be exposing a simpler property for the view to consume so that it doesn't have to add it's own logic to find the data it needs. For instance, this could be:

```swift
ForEach(viewModel.ongoingMeetings) { meeting in 
  // ..
}
```

And in the view model, `ongoingMeetings` is using the repository and determining which of the ongoing meetings it exposes to the view.